src/main/groovy/de/dkfz/roddy/execution/jobs/BEJob.groovy

```diff
@@ -130,12 +124,30 @@ class BEJob<J extends BEJob, JR extends BEJobResult> {
         this.toolMD5 = toolMD5
         this.resourceSet = resourceSet
         this.parameters = parameters
-        this.parentJobs = parentJobs
-        this.arrayIndices = arrayIndices ?: new LinkedList<String>()
-        this.listOfCustomDependencyIDs.addAll(parentJobIDs ? parentJobIDs : new LinkedList<BEJobID>())
         this.jobManager = jobManager
     }
 
+    static BEJob fromJobID(BEJobID jobID, BatchEuphoriaJobManager jobManager = jobID?.job?.jobManager) {
```

Comment:
O.k. using the constructor is not the appropriate way. The issue is that we return the job that is referenced by the jobID, if possible. With a constructor we cannot do that.

Do we want to stick to the returning of the original job? I think it has advantages (in particular, no copy constructor necessary).